File: mailpython/enviar_correo.py
import smtplib
from email.mime.text import MIMEText
from email.mime.multipart import MIMEMultipart

# Credenciales
remitente = "jesusernestoortiz405@gmail.com"
contrasena_app = "vnijgnhgnhhuurix"  # Contraseña de aplicación (sin espacios)
destinatario = "jesusernestoortiz405@gmail.com"

# Crear mensaje
mensaje = MIMEMultipart()
mensaje["From"] = remitente
mensaje["To"] = destinatario
mensaje["Subject"] = "Correo de prueba con Python"

cuerpo = "Hola, este correo fue enviado desde Python puro usando Gmail y contraseña de aplicación."
mensaje.attach(MIMEText(cuerpo, "plain"))

try:
   # Configurar servidor SMTP
   servidor = smtplib.SMTP("smtp.gmail.com", 587)
   servidor.starttls()
   servidor.login(remitente, contrasena_app)
   servidor.sendmail(remitente, destinatario, mensaje.as_string())
   servidor.quit()
   print("✅ Correo enviado correctamente.")
except Exception as e:
   print("❌ Error al enviar el correo:", e)

File: mailserver/phpMailer/enviar_correo.php
<?php
// Incluir las clases necesarias de PHPMailer
require     'PHPMailer-master\PHPMailer-master\src\PHPMailer.php';
require     'PHPMailer-master\PHPMailer-master\src\SMTP.php';
require     'PHPMailer-master\PHPMailer-master\src\Exception.php';

// Usar los namespaces de PHPMailer
use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\Exception;

// Crear una nueva instancia de PHPMailer
$mail = new PHPMailer(true);

try {
    // Configuración del servidor SMTP
    $mail->isSMTP(); // Usar SMTP
    $mail->Host = 'smtp.gmail.com'; // Servidor SMTP (en este caso, Gmail)
    $mail->SMTPAuth = true; // Habilitar autenticación SMTP
    $mail->Username = 'jesusernestoortiz405@gmail.com'; // Dirección de correo del remitente
    $mail->Password = 'vnijgnhgnhhuurix'; // Contraseña o clave de aplicación del correo
    $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS; // Tipo de encriptación (TLS)
    $mail->Port = 587; // Puerto SMTP para TLS

    // Configuración del remitente y destinatario
    $mail->setFrom('jesusernestoortiz405@gmail.com', '104 CUBES'); // Dirección y nombre del remitente
    $mail->addAddress('jesusernestoortiz405@gmail.com', 'Jesus Ernesto Ortiz'); // Dirección y nombre del destinatario

    // Configuración del contenido del correo
    $mail->isHTML(true); // Indicar que el contenido es HTML
    $mail->Subject = 'Asunto del correo'; // Asunto del correo
    $mail->Body = 'Este es el cuerpo del correo.'; // Cuerpo del correo en formato HTML
    $mail->AltBody = 'Este es el cuerpo del correo en texto plano.'; // Cuerpo del correo en texto plano (opcional)

    // Enviar el correo
    $mail->send();
    echo '<div style="color: green; font-weight: bold; font-family: Arial, sans-serif; margin-top: 20px;">El mensaje se envió correctamente.</div>'; // Mensaje de éxito
} catch (Exception $e) {
    // Manejo de errores
    echo "Hubo un error al enviar el mensaje: " . htmlspecialchars($mail->ErrorInfo); // Mostrar el error de forma segura
}
